### PR TITLE
Fixed code review items. Details are in the description

### PR DESCRIPTION
**Pull request by genaimagician**
1)For all 7 notebooks, I added copyright wordings, rewrote titles, subtitles, and added extra descriptions and instructions . 
2)Remove demo folder
3)Rewrote readme 
4)Updated resources for setting up search engines, vector store, and cloud functions
5)Checked grammars and spellings using Pycharm spelling plugins

### CODE REVIEW

File: genai-on-vertex-ai/developer_productivity_with_genai/utilities/webhook_cloud_function.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
+from __future__ import annotations
```
Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/utilities/webhook_cloud_function.py:1]: Please add copyright wordings.

**Review by RajeshThallam — COMMENTED** [review]
Does this need to be added to the repo? Please ignore if it has to be a zip file.

genai-on-vertex-ai/developer_productivity_with_genai/utilities/bank_of_anthos_codebase.zip

File: genai-on-vertex-ai/developer_productivity_with_genai/README.md [hunk 2]
```diff
@@ -1,75 +1,22 @@
...
-### wireframe to live website notebook covers 
-User Journey - Code Generation support for more languages (HTML , CSS, Objective-C etc):
+We offer a comprehensive set of notebooks that demonstrate how to use Codey APIs to solve various tasks in software development life cycle including code generation, unit test generation, code explanation, comment generation, code refactoring, debugging, migration, codebase chat, Doc search, and JIRA search.
 
-Tech Tasks:
-- Generate functions
-- Explain code
-- Generate unit tests
-- Assisted code refactoring and modification
+Notebooks:
```
Comment (RajeshThallam) [inline genai-on-vertex-ai/developer_productivity_with_genai/README.md:7]: Please add links to the notebooks for better navigation.

File: genai-on-vertex-ai/developer_productivity_with_genai/utilities/webhook_cloud_function.py (hunk 1 above)
Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/utilities/webhook_cloud_function.py:16]: Done. Please check.

File: genai-on-vertex-ai/developer_productivity_with_genai/README.md (hunk 2 above)
Comment (genaimagician) [reply genai-on-vertex-ai/developer_productivity_with_genai/README.md:7]: Done. Please check.

**Comment by genaimagician** [comment]
> Does this need to be added to the repo? Please ignore if it has to be a zip file.
> 
> genai-on-vertex-ai/developer_productivity_with_genai/utilities/bank_of_anthos_codebase.zip

For this one, it needs to be a zip because this is exactly the codebase I used in the vector store. If users want to replicate exactly what I did, they need to upload this codebase to a bucket and chunk them from there. 

This repo is from a GCP open source project - bank of anthos. The reason I didn't point users to that public repo is because the original code repo failed many times when I chunked them to embeddings and I fixed a few things to make it work. This zip has all the fixes I did.

**Review by RajeshThallam — APPROVED** [review]
Thanks for resolving the comments. LGTM.

**Pull request merged**